README.md

```diff
@@ -133,13 +134,14 @@ Install this project with:
 
 ## 👥 Authors <a name="authors"></a>
 
```

Comment:
- [ ] Please since you are the only contributor to this project you can change the `Authors` section to `Author` :pray: